languages/translations/fr.json

```diff
@@ -41,6 +41,9 @@
                 "ox": "Serrer à gauche",
                 "xox": "Rester au milieu",
                 "oxo": "Rester à gauche ou à droite"
+            },
+            "vocabulary": {
+              "exit": "la sortie"
```

Comment:
Is this term being used as part of an instruction or in a standalone context, such as in banner text in the navigation SDKs? This library already incorporates the words “the exit” into some instruction formats:

https://github.com/Project-OSRM/osrm-text-instructions/blob/895a20330f017d6ad3ed6ec983dbcb1eba331ee2/languages/translations/fr.json#L299